app/src/main/res/layout/view_allpatientsinfacility.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.allpatientsinfacility.AllPatientsInFacilityView xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:animateLayoutChanges="true">
```

Comment:
I'm not sure if `animateLayoutChanges` is advisable here because of recyclerview view changes too. Need to confirm this.

Reply:
I am unsure if this problem can happen if `animateLayoutChanges` is set on the `RecyclerView` itself or if it can happen if the property is set on the parent itself.